lib/libutee/arch/arm/utee_syscalls_asm.S

```diff
@@ -121,47 +121,49 @@
         UTEE_SYSCALL utee_cryp_obj_generate_key, \
                      TEE_SCN_CRYP_OBJ_GENERATE_KEY, 4
 
-        UTEE_SYSCALL utee_se_service_open, TEE_SCN_SE_SERVICE_OPEN, 1
+        UTEE_SYSCALL utee_se_service_open, \
```

Comment:
Do we need to keep these entries?
Especially since `lib/libutee/tee_api_se.c` is removed.

Reply:
Nope, they are totally useless indeed. WIll remove them.